### DISCUSSION TITLE
Feature Request: Option to get timestamp of files when reading globs

### DISCUSSION THREAD

**Discussion by brendonparker** [duckdb/duckdb#14700] [Ideas]
Requesting an option for `read_csv` that would including the timestamp of the file creation and/or last modified would be extremely helpful. Especially in the scenario where files are write-only to S3 for a form of snapshot/archiving, so that you can get the timestamp along with the records.

Proposed usage:
```sql
SELECT FlightDate, OriginCityName, DestCityName, UniqueCarrier, filename, last_modified
FROM read_csv(['flights1.csv', 'flights2.csv'], filename = true, last_modified = true);
```

or with globbing
```sql
SELECT FlightDate, OriginCityName, DestCityName, UniqueCarrier, filename, last_modified
FROM read_csv('s3://bucket/**/flights.csv', filename = true, last_modified = true);
```

**Comment by jimdickson007** [comment]
In a similar vein, I would like a checksum for each file.
https://github.com/duckdb/duckdb/discussions/13936
